app/modules/main/templates/mdDataTableColumn.html

```diff
@@ -1,13 +1,12 @@
-<th class="column {{columnAlignClass}}" ng-click="clickHandler()">
+<th class="column {{columnAlignClass}}" ng-disabled="!isSortingEnabled()" md-ink-ripple ng-click="clickHandler()">
     <div layout="row">
         <span ng-if="isSortingEnabled() && isColumnRightAligned()">
```

Comment:
since `ng-disabled` checking the `isSortEnabled()` option, I'd remove this `isSortingEnabled()` from here.
So you'll have only `ng-if=isColumnRightAligned()`